From 15c7a38af3d6c157cd8db07043de274cdf97648f Mon Sep 17 00:00:00 2001
From: Jose Lamego <jose.a.lamego@intel.com>
Date: Thu, 23 Aug 2018 11:47:30 -0500
Subject: [PATCH 2/2] add default signature verification policy file and
 registries conf

This commit creates a default signature verification policy file
file to allow image and signature validations.

Fixes clearlinux/distribution#160

Signed-off-by: Jose Lamego <jose.a.lamego@intel.com>
---
 policy.json     | 11 +++++++++++
 registries.conf |  1 +
 2 files changed, 12 insertions(+)
 create mode 100644 policy.json
 create mode 100644 registries.conf

diff --git a/policy.json b/policy.json
new file mode 100644
index 0000000..fc643c8
--- /dev/null
+++ b/policy.json
@@ -0,0 +1,11 @@
+{
+    "default": [
+        {
+            "type": "insecureAcceptAnything"
+        }
+    ],
+    "transports":
+        {
+            "docker": {}
+        }
+}
diff --git a/registries.conf b/registries.conf
new file mode 100644
index 0000000..c2760f2
--- /dev/null
+++ b/registries.conf
@@ -0,0 +1 @@
+unqualified-search-registries = ['quay.io' ,'registry.access.redhat.com', 'registry.fedoraproject.org', 'ghrc.io', 'docker.io']
-- 
2.47.1

